core-chart/templates/postcreatehooks/its-clusteradmin.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.InstallPCHs }}
+apiVersion: tenancy.kflex.kubestellar.org/v1alpha1
+kind: PostCreateHook
+metadata:
+  name: its-clusteradmin
```

Comment:
"its-clusteradmin" is at best not really informative, at worst misleading. This hook is not about administration in general. It is specifically about invoking `clusteradm init` to initialize the cluster as an OCM hub. Perhaps a name like "its-hub-init"?